services/gitea/9.3.0/defaults/pvc.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# In the previous versions (v8.x and prior), PVC was automatically managed by the Statefulset, while in the
+# new versions (v9.x and later), it is automatically handled by the Helm chart.
+# In order to remain backward compatible with previous installations, we manually create a PVC object,
+# filling in its name value with the data-gitea-0 that is the value in gitea v8.x and older versions.
+kind: PersistentVolumeClaim
```

Comment:
will flux be able to deploy this resource 🤔 ? does it complain that there is already a resource deployed with this name?

Reply:
Would need to set create pvc to false too I think https://gitea.com/gitea/helm-chart/src/branch/main/templates/gitea/pvc.yaml#L1

Reply:
PVC is only created for new installations. For upgrade processes, we keep the existing PVC.

```yaml
    persistence:
      enabled: true
      create: false
      mount: true
      claimName: "data-gitea-0"
```